src/App.tsx

```diff
@@ -25,12 +27,13 @@ export const App = () => (
     <NavHeader />
     <Routes>
       <Route path="/" element={<Home />} />
-      <Route path="/login" element={<Login />} />
+      <Route path="/login" element={<Login />} /> 
       <Route path="/generate" element={<Generate />} />
       <Route path="/schedule" element={<Schedule />} />
       <Route path="/surveyresults" element={<SurveyResults />} />
       <Route path="/dashboard" element={<Dashboard />} />
       <Route path="/profileManagement" element={<ProfileManagement />} />
+      <Route path="/professorSurvey" element={<ProfessorSurvey />} />
```

Comment:
With Jaydens header navbar changes, the link in the nav bar routes to /survey but the route here links to /professorSurvey. Idk which one we want to stick with but we should make sure these match. Right now the header link 404's.